core/src/main/scala/org/http4s/internal/fs2.scala

```diff
@@ -0,0 +1,27 @@
+package org.http4s.internal
+
+import _root_.fs2.io.tls.TLSParameters
+import javax.net.ssl.SSLParameters
+import org.http4s.internal.CollectionCompat.CollectionConverters._
+
+private[http4s] object fs2 {
+  def toSSLParameters(tlsParameters: TLSParameters): SSLParameters = {
+    val p = new SSLParameters()
+    import tlsParameters._
+    algorithmConstraints.foreach(p.setAlgorithmConstraints)
+    // applicationProtocols.foreach(ap => p.setApplicationProtocols(ap.toArray))
```

Comment:
If we leave these commented, they get ignored on JDK11 despite the `TLSParameters` type implying they'll be respected.  If we support them, we have to start building on JDK11 and risk accidentally calling its methods while claiming to be a JDK8 library.

If we can get https://github.com/functional-streams-for-scala/fs2/pull/1768 released, then on fs2's heads be it.

Reply:
We can live with this as is, because we're expecting an fs2 release that publicizes this.